R/evaluate-clusters.R

```diff
@@ -307,3 +307,125 @@ calculate_stability <- function(
 
   return(all_ari_df)
 }
+
+#' Evaluate cluster results
+#'
+#' This wrapper function can be used to evaluate clusters from a single clustering calculation or a list
+#' calculated using `sweep_clusters()` function.
+#' Input should be be a data frame from a single `calculate_clusters()` call
+#' or a list of data frames with the resulting clusters from all parameter
+#' combinations provided to the `sweep_clusters()` function.
+#' Output is a list of results or a single data.frame depending on what was provided.
+#' Evaluation statistics are added in the form of columns to the original cluster
+#' object provided.
+#'
+#' @param x An object containing PCs that clusters were calculated from. This can be
+#'   either a SingleCellExperiment object, a Seurat object, or a matrix where columns
+#'   are PCs and rows are cells. If a matrix is provided, it must have row names of cell
+#'   ids (e.g., barcodes).
+#' @param cluster_results A single data frame or list of data frames obtained from
+#'   `rOpenScPCA::sweep_clusters()`. Each data frame in the list should contains
```

Comment:
Here is where I would say that the single data frame is typically from `rOpenScPCA::calculate_clusters()`.